docs/deploying.md

```diff
@@ -128,95 +128,85 @@ This snippet assumes the following resources are already defined:
 
 Steps to deploy:
 
-1. Paste your VPC ID, public subnet ID, and CIDR block into the parameters. Change the [ImageId](index.md#getting-a-fck-nat-ami) based on the region fck-nat is deployed to.
+1. Paste your VPC ID, public subnet ID, VPC CIDR block into the parameters. Change the [FckNatAMIParameter](index.md#getting-a-fck-nat-ami) based on the region fck-nat is deployed to.
 2. Ensure that your public subnet has `Enable auto-assign public IPv4 address` turned on. This can be found in the Console at `VPC > Subnets > Edit subnet settings > Auto-assign IP settings`.
-3. Deploy with cloudformation `aws cloudformation deploy --force-upload --capabilities CAPABILITY_IAM --template-file template.yml --stack-name FckNat`
-4. Add the default route to your route table on the subnet. It is best to do this manually so you can do a seamless cut over from your existing nat gateway. Go to `VPC > Route Tables > Private route table > Routes > Edit Routes` Add a 0.0.0.0/0 route pointing to the network interface.
+3. Deploy with CloudFormation `aws cloudformation deploy --force-upload --capabilities CAPABILITY_IAM --template-file template.yml --stack-name FckNat`
+4. Add the default route to your route table on the subnet. It is best to do this manually so you can do a seamless cut over from your existing NAT gateway. Go to `VPC > Route Tables > Private route table > Routes > Edit Routes` Add a 0.0.0.0/0 route pointing to the network interface.
 
 ``` yaml
 Parameters:
-  vpc:
-    Type: String
-    Default: "vpc-121212121212121212"
-  subnet:
-    Type: String
-    Default: "subnet-121212121212121212"
-  CIDR:
+  VpcIdParameter:
+    Type: AWS::EC2::VPC::Id
+  SubnetIdParameter:
+    Type: AWS::EC2::Subnet::Id
+  CIDRParameter:
     Type: String
     Default: "10.0.0.0/16"
+  FckNatAMIParameter:
+    Type: AWS::EC2::Image::Id
+    Default: ami-075a0093cd9926d44
```

Comment:
I need to double check this AMI ID

Reply:
Maybe it would be better to not have a default here and just have the description point at the CLI command to run? Any default would be region-specific and would get outdated quickly.

Reply:
Yeah, I think that's best. This conversation has happened in PRs a few times now.